lib/mix/nerves/shell.ex

```diff
@@ -68,14 +78,20 @@ defmodule Mix.Nerves.Shell do
     end
   end
 
-  @ctrl_op_get_winsize 100
```

Comment:
This is now 0x018b0900 bigger. See https://github.com/erlang/otp/commit/ad5822c6b1401111bbdbc5e77fe097a3f1b2b3cb

Reply:
wow, good find thanks!